packages/airnode-node/src/providers/state.ts

```diff
@@ -16,7 +16,11 @@ export function buildEVMState(
   const masterHDNode = evm.getMasterHDNode(config);
   const chainProviderUrl = chain.providers[chainProviderName].url;
   const provider = evm.buildEVMProvider(chainProviderUrl, chain.id);
-  const airnodeAddress = evm.getAirnodeWallet(config).address;
+  const airnodeWalletPrivateKey = getEnvValue('AIRNODE_WALLET_PRIVATE_KEY');
+  if (!airnodeWalletPrivateKey) {
+    throw new Error('Missing Airnode wallet private key in environment variables.');
+  }
+  const airnodeAddress = evm.getAirnodeWalletWithPrivateKey(airnodeWalletPrivateKey).address;
```

Comment:
Can't we pass the `airnodeAddress` to the `buildEVMState` function. It should be already available level higher, isn't it?

Reply:
You're right, its available in CoordinatorSettings. I'll update